src/cf-typography/usage.md

```diff
@@ -1,30 +1,93 @@
+The cf-typography component includes multiple patterns for headings, links and other typographical patterns.
```

Comment:
"The cf-typography component includes multiple patterns for headings, links, lists and other advanced typographic treatments."